tableschema/exceptions.py

```diff
@@ -44,10 +44,18 @@ class IntegrityError(TableSchemaException):
     pass
 
 
+class UniqueKeyError(CastError):
+    pass
+
+
 class RelationError(TableSchemaException):
     pass
 
 
+class UnresolvedFKError(RelationError):
```

Comment:
Is it a subcase for `RelationError`?

If yes, please add to `README.md#exceptions`

Reply:
> the exception specializations for FK and unique key errors come in handy for nicer error reporting and better distinguishable errors imho

I see

Reply:
Yes it's a subclass of RelationError (to remain compatible) and it's intention is to make it distinguishable from Unique Key violations. Both for error reporting and also because it comes in handy to build upon in datapackage (you can then use it to not report/raise Unique Key violations of a referenced resource for the referencing resource).

Reply:
Documentation added.